fare/records/config.py

```diff
@@ -46,10 +47,10 @@ def _(x):
         default_media_type='application/json',
         max_result_window=10000,
         error_handlers=dict(),
-        create_permission_factory_imp=allow_all,
+        create_permission_factory_imp=deny_all,
```

Comment:
Could you please explain me why we need a `deny_all` rule? Isn't this clashing with the permission system in place? Does this mean that if I create a RESTful client and try to connect to this server I will get a 500? I would like to have a functioning RESTful server so this `deny_all` is kinda scary (but maybe there is a logic behind it!). Thanks!

Reply:
With the `allow_all` permission every users can create a record composed only with its metadata and no content attached to it, that is not what we want. In order to do so @CorsoGabriele and I have created the view in the file_management module, in this way we can also check if the user has the correct permission to create a record. Now we are working to create the test for this part

Reply:
Ok, great